pre-commit.md

```diff
@@ -0,0 +1,16 @@
+# Setting up Pre-Commit
+
+When creating a new empty repository, please add these configuration files into the repository
```

Comment:
Should we point out here that for a Python-based repository, using the template is the preferred approach?